tests/Unit/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Test_BjorhusImpl.cpp

```diff
@@ -336,7 +336,7 @@ void test_constraint_preserving_bjorhus_v_zero_vs_spec_3d(
   CHECK_ITERABLE_APPROX(local_bc_dt_v_zero, spec_bc_dt_v_zero);
```

Comment:
The clang-tidy complaint about this file is legit. Could you fix the include grouping?